src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -143,6 +186,8 @@ def __init__(
         dynamic_thresholding_ratio: float = 0.995,
         clip_sample_range: float = 1.0,
         sample_max_value: float = 1.0,
+        timestep_spacing: str = "leading",
+        rescale_betas_zero_snr: bool = False,
```

Comment:
@patrickvonplaten this is for my understanding. 

Do we not need to register these variables?

Reply:
The `register_to_config` decorator automatically does this